spec/rubyhub/configuration_spec.rb

```diff
@@ -10,6 +12,44 @@
     end
   end
 
+  describe '#initialize' do
+    context 'when options' do
+      before do
+        Singleton.__init__(described_class)
+      end
+
+      it 'not loaded from file' do
```

Comment:
nit: here should be context 'when options not loaded from file', and `it` block should be like `returns that` or `do that`.